deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd $(dirname "$BASH_SOURCE")
```

Comment:
I understand the first part of your comment even if I am not sure it should be a concern here.

> this is mostly useful in the context of sourced scripts (as opposed to executed ones).
Can you clarify your comment or give me some pointers ? Thanks !